src/Reflex/Dom/ACE.hs

```diff
@@ -143,233 +141,267 @@ data AceDynConfig = AceDynConfig
 instance Default AceConfig where
     def = AceConfig def def def False False
 
-#ifndef ghcjs_HOST_OS
-data Element = Element
-data JSVal = JSVal
-jsNull :: JSVal
-jsNull = JSVal
-
-jsval :: a -> JSVal
-jsval = const JSVal
-
-toJSString :: a -> b
-toJSString _ = undefined
-
-#endif
-
-newtype AceRef = AceRef { unAceRef :: JSVal }
+newtype AceInstance = AceInstance { unAceInstance :: JSVal }
 
 data ACE t = ACE
```

Comment:
Couldn't you avoid this by keeping the old name `AceRef` instead of switching to `AceInstance`?